test/verify/check-system-services

```diff
@@ -789,6 +789,24 @@ Where=/fail
 
         self.allow_journal_messages(".*type=1400 audit(.*): avc:  denied  { create } .* comm=\"systemd\" name=\"fail\".*")
 
+    def testTransientFailingUnits(self):
+        self.login_and_go("/system/services")
+        self.wait_page_load()
+
+        self.machine.execute("systemd-run --collect --unit test-autocollect@1.service sh -c 'sleep 3; false'")
+        self.machine.execute("systemd-run --unit test-manual-collect@1.service sh -c 'sleep 3; false'")
+
+        self.wait_service_present("test-autocollect@1.service")
+        self.wait_service_present("test-manual-collect@1.service")
+
+        self.wait_service_in_panel("test-manual-collect@1.service", "Failed to start")
+
+        self.wait_service_not_present("test-autocollect@1.service")
```

Comment:
This is a race condition -- it would be more robust to replace the `sleep 3` with something like `until test -e /tmp/autocollect-continue; do sleep 0.5; done` and touch the file here. But tests seem happy, we can still do that when necessary.